pkg/virtctl/create/vm/vm.go

```diff
@@ -131,21 +140,10 @@ type createVM struct {
 	memoryChanged                 bool
 
 	clientConfig clientcmd.ClientConfig
+	cmd          *cobra.Command
```

Comment:
This will create a circular reference: `createVM` struct contains `cobra.Command` and `cobra.Command` contains `createVM.run` method. It is not a problem, but I find it confusing.

Reply:
Indeed looks a bit confusing. But the method/function set to `RunE` gets the `cmd` passed as argument. To me this means code run by the `cobra.Command` should be aware of the command and to access it. (e.g. to access flags or to print something)

Reply:
ok